scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -187,18 +192,32 @@ def net_interfaces_config(args, nicmodel):
 
 def net_config(args):
     cloud = args.cloud
-    values = dict(
-        cloud=cloud,
-        cloudbr=args.cloudbr,
-        admingw=args.admingw,
-        adminnetmask=args.adminnetmask,
-        cloudfqdn=args.cloudfqdn,
-        adminip=args.adminip,
-        forwardmode=args.forwardmode)
+    values = {
+        'cloud': cloud,
+        'cloudbr': args.cloudbr,
+        'admingw': args.admingw,
+        'adminnetmask': args.adminnetmask,
+        'cloudfqdn': args.cloudfqdn,
+        'adminip': args.adminip,
+        'forwardmode': args.forwardmode
+    }
 
     return get_config(values, os.path.join(TEMPLATE_DIR, "admin-net.xml"))
 
 
+def ironic_net_config(args):
```

Comment:
this looks quite similar to `net_config`, so I wonder if both use cases could use the same function that supports both cases